### DISCUSSION TITLE
Swissup breeze implementation

### DISCUSSION THREAD

**Discussion by bmandl** [Smile-SA/elasticsuite#2289] [Q&A]
Hello guys. Let me first give you respect for such a quality work on opensource community. This module really looks proffesional. I am long looking for some module for Magento 2, even tried to build it myself, but am still strugling with, to be able to select multiple filter attributes of same group in layered navigation and join these attributes with OR conditions and groups with AND conditions. There are tons of extensions, that do something similar, but I still haven't found one that would done a good job. I have a feeling, that elasticsuite would be perfect for the job, but the thing is, I am using Swissup breeze module with my Magento: [breeze](https://swissuplabs.com/magento-2-breeze-module.html)
Would there be anyone interested, to help convert me existing Javascript in elasticsuite to work with Breeze? Just for catalog module at first?
Have a nice day.

**Comment by dimdimich** [comment]
Hi, it's already added at https://github.com/breezefront/module-breeze-smile-elasticsuite by Breeze team.
